mne/decoding/receptive_field.py

```diff
@@ -0,0 +1,353 @@
+import numpy as np
+from .base import _get_final_est, BaseEstimator, _check_estimator
+
+
+class ReceptiveField(BaseEstimator):
```

Comment:
I think somebody told me to put them in `fit` earlier on so they behaved like other sklearn objects. But you're right in that the attributes could be created upon init. maybe @kingjr has thoughts on this, but I'm happy to move them back to `init` and remove the trailing underscore.